src/components/Footer/index.tsx

```diff
@@ -30,35 +52,21 @@ export default function DocsFooter() {
               <img alt={img.GITHUB.alt} src={img.GITHUB.lightSrc} />
             </ExternalLink>
           </Social>
-          <Legal>
-            <span>
-              <div>
-                <p>
-                  <img alt={img.AWS.alt} src={img.AWS.lightSrc} />
-                  Amplify open source software, documentation and
-                  <br /> community are supported by Amazon Web Services.
-                </p>
-                <p>
-                  © {new Date().getFullYear()}, Amazon Web Services, Inc. and
-                  its affiliates.
-                </p>
-                <p>
-                  All rights reserved. View the{' '}
-                  <ExternalLink href={links.TERMS}>site terms</ExternalLink> and{' '}
-                  <ExternalLink href={links.PRIVACY}>
-                    privacy policy
-                  </ExternalLink>
-                  .
-                </p>
-              </div>
-              <div className="margin-top-md">
-                Flutter and the related logo are trademarks of Google LLC. We
-                are not endorsed by or affiliated with Google LLC.
-              </div>
-            </span>
-          </Legal>
+          <div>
+            <div>We want to hear from you</div>
+            <Link href="https://docs.amplify.aws">Take our docs survey</Link>
```

Comment:
neither of these links take the user to where they would expect.  I know we don't have a survey currently available so can we remove this until we do?  Also should connect to an expert actually point to the Amplify discord and be using an <ExternalLink> instead of just adding the icon?